server/models/Query.ts

```diff
@@ -6,6 +6,7 @@ const querySchema = new Schema({
     number: { type: Number, unique: true, required: true },
     depth: { type: Number, required: false },
     complexity: { type: Number, required: false },
+    success: { type: Boolean, required: true },
     timestamp: { type: Number, required: false },
     logged_on: { type: Date },
```

Comment:
Are `logged_on` and `timestamp` the same thing? Or is this the beginning and end of the query to calculate `latency`?